### DISCUSSION TITLE
The history of Squirrel.Windows

### DISCUSSION THREAD

**Discussion by vernou** [Squirrel/Squirrel.Windows#1768] [General]
Hi,

I have a some questions that is related to the history of Squirrel.Windows :

- Who initiated the project and for what purpose? (GitHub to deploy/update electron apps?)
- Why the choice of C++ and .NET? (No dependency required because integrated in Windows?)
- Who manage own/manage the project? (GitHub, Microsoft, @anaisbetts, the community, ...?)

If an elder is willing to tell us a tale, I'm all ears.

**Comment by r-pankevicius** [comment]
https://github.com/Squirrel/Squirrel.Windows/commits/develop?after=391b19099d9524eaab038c2d5e50be422748c305+209&branch=develop&qualified_name=refs%2Fheads%2Fdevelop
